manifests/params.pp

```diff
@@ -107,6 +112,8 @@
   $rhn_password   = '',
   $genid_command  = '/usr/bin/gensystemid',
   $mailto         = 'UNSET',
+  $mailfrom       = 'UNSET',
```

Comment:
@jearls It might be worth taking the opportunity to refactor and default to `undef` instead of `'UNSET'`?

The existing code is pretty old I think.  I'm not sure the following was ever required (like in puppet 2??), but
```ruby
<% unless (mailto = @mailto) == 'UNSET' -%>
  mailto = <%= mailto %>
<% end -%>
```
is pretty ugly.
```ruby
<% if @mailto -%>
  mailto = <%= @mailto %>
<% end -%>
```
would be much nicer I think.